corehq/motech/repeaters/tasks.py

```diff
@@ -201,14 +204,15 @@ def _process_repeat_record(repeat_record):
 )
 
 
-@task(serializer='pickle', queue=settings.CELERY_REPEAT_RECORD_QUEUE)
-def process_repeater(repeater: SQLRepeater):
+@task(queue=settings.CELERY_REPEAT_RECORD_QUEUE)
+def process_repeater(repeater_id):
```

Comment:
I'll happily continue using typing here, just wasn't sure what the id would be? Is it just an `int`?

Reply:
Yeah, it's a normal int Django model primary key. ... And a type hint would be great, to avoid any confusion with `SQLRepeater.repeater_id`, which is the UUID of a Couch Repeater.